src/wallet/types.ts

```diff
@@ -20,6 +21,11 @@ export type ConnectWalletReact = {
   /** @deprecated Prefer `ConnectWalletText component` */
   text?: string; // Optional text override for button
   withWalletAggregator?: boolean; // Optional flag to enable the wallet aggregator like RainbowKit
+  /**
+   * Optional callback function to execute when the wallet is connected.
+   * This can be used to perform additional actions after the wallet is connected.
+   */
+  onInitialConnect?: () => void;
```

Comment:
nit: let's keep this comment inline with `onInitialConnect`, similar to the other comments in this file

Reply:
Fair enough, updated.

BTW the reason for using multiline comments is that the comment then shows up in Intellisense. I think this helps improve DX (internally and externally) but I understand that we already have a convention around `//` style comments.
  
![CleanShot 2024-10-30 at 08 09 03@2x](https://github.com/user-attachments/assets/cd8ab7bc-6da0-48ed-a59b-92026a6d0861)